doc/how_to/migrate/from_streamlit.md

```diff
@@ -0,0 +1,947 @@
+).servable(target="main")
+
+pn.state.template.param.update(
+    site="Panel",
+    title="Template Example",
+    header_background="#E91E63",
+    accent_base_color="#E91E63",
+)
+```
+
+The app looks like
+
+![Panel Template Example](https://assets.holoviz.org/panel/gifs/panel_app_example.gif)
+
+### Template Migration Steps
+
+When migrating you will have to decide which template to use
+
+- Blank (default)
+- A built in like *vanilla*, *bootstrap*, *material* or *fast*. See the
+[Templates Section](../../reference/index#templates) of the
+[Components Guide](../../reference/index).
+- A custom template
+
+## Accepting User Inputs with Widgets
+
+In Panel the objects that can accept user inputs are called *widgets*.
+
+Panel provides widgets similar to the ones you know from Streamlit and some unique ones in addition.
+
+Check out the
+[Widgets Section](../../reference/index.md#widgets) of
+the [Component Gallery](../../reference/index.md) for the full list of
+*widgets*.
+
+### Streamlit Integer Slider Example
+
+```python
+import streamlit as st
+
+bins = st.slider(value=20, min_value=10, max_value=30, step=1, label="Bins")
+
+st.write(bins)
+```
+
+The app looks like
+
+![Streamlit Widgets Example](../../_static/images/streamlit_widgets_example.png)
+
+### Panel Integer Slider Example
+
+You will find Panels input *widgets* in `pn.widgets` module.
+
+```python
+import panel as pn
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+bins = pn.widgets.IntSlider(value=20, start=10, end=30, step=1, name="Bins")
+
+pn.Column(bins, pn.pane.Str(bins)).servable()
+```
+
+If you check the type of the variables, you will notice a key difference. Streamlit's `bins` returns the value of the slider as an `integer` while Panel's `bins` returns an `IntSlider`!
+
+To access the value of the slider in Panel, you would need to call `bins.value`.
+
+For more info about the `IntSlider` check out the
+[`IntSlider` Guide](../../reference/widgets/IntSlider.md).
+
+The app looks like
+
+![Panel Widgets Example](../../_static/images/panel_widgets_example.png)
+
+### Input Migration Steps
+
+You should
+
+- replace your Streamlit `st.some_widget` function with the corresponding Panel
+`pn.widgets.SomeWidget` class.
+
+You can identify the corresponding widget via the
+[Widgets Section](../../reference/index.md#widgets) of the [Component Gallery](../../reference/index.md).
+
+## Show Activity
+
+Panel supports many ways of indicating activity
+
+- Indicators. See the [Indicators Section](../../reference/index.md#indicators)
+of the [Component Gallery](../../reference/index.md).
+- `disabled`/ `loading` parameters on Panel components
+- `loading_indicator` parameter for `pn.panel` or `pn.config`. If `True` a
+loading indicator will be shown on your *bound functions* when they are re-run.
+
+```python
+import panel as pn
+
+pn.extension(sizing_mode="stretch_width", template="bootstrap")
+
+SPIN_CSS = """
```

Comment:
Being able to transform rotate icons could be added to Panel one day @philippjfr ?